fnet/cli/init.py

```diff
@@ -39,12 +44,11 @@ def save_default_train_options(path_save: str) -> None:
         Save path for default training options json.
 
     """
-    if os.path.exists(path_save):
-        print('Training options file already exists:', path_save)
+    path_save = Path(path_save)
+    if path_save.exists():
+        logger.info(f'Training options file already exists: {path_save}')
```

Comment:
I'd probably throw  a custom exception here personally.

Reply:
An exception would imply that something went "wrong" though. The idea is to simply exit if the intended action had already been completed in the past. That way the user does not have to manually check the file system.